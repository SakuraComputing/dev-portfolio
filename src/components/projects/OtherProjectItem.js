import React from 'react';

const OtherProjectItem = (props) => {
    return (
        <div>
            <div className="item">
                <h3 className="title"><a href="#">{props.otherproject.title}</a> <span
                    className="label label-theme">Open Source</span></h3>
                <p className="summary">{props.otherproject.description}</p>
                <p><a className="more-link"
                      href={props.otherproject.github}
                      target="_blank"><i className="fa fa-external-link"></i> View on GitHub</a></p>
            </div>
        </div>
    );
};

export default OtherProjectItem;
