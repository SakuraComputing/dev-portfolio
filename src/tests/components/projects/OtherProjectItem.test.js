import React from 'react';
import { shallow } from 'enzyme';
import OtherProjectItem from '../../../components/projects/OtherProjectItem';

describe('Other Projects Section', () => {
    it('should render correctly', () => {
        const otherproject = jest.fn();
        const wrapper = shallow(<OtherProjectItem otherproject={otherproject}/>);
        expect(wrapper).toMatchSnapshot();
    });
});