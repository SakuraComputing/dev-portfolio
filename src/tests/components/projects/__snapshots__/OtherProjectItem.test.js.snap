// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Other Projects Section should render correctly 1`] = `
<div>
  <div
    className="item"
  >
    <h3
      className="title"
    >
      <a
        href="#"
      />
       
      <span
        className="label label-theme"
      >
        Open Source
      </span>
    </h3>
    <p
      className="summary"
    />
    <p>
      <a
        className="more-link"
        href="https://github.com/SakuraComputing/embDocs"
        target="_blank"
      >
        <i
          className="fa fa-external-link"
        />
         View on GitHub
      </a>
    </p>
  </div>
</div>
`;
